decidim-core/app/packs/src/decidim/index.js

```diff
@@ -122,4 +121,14 @@ $(() => {
   markAsReadNotifications()
 
   scrollToLastChild()
-});
+}
+
+if ("Turbo" in window) {
+  document.addEventListener("turbo:frame-render", () => initializer());
+  document.addEventListener("turbo:load", () => initializer());
+} else {
+  // If no jQuery is used the Tribute feature used in comments to autocomplete
+  // mentions stops working
+  // document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Why this comment @entantoencuanto ?

Reply:
I put it there when I tried to get rid of jQuery markup a time ago. I left the vanilla-js for reference about what should be the actual event